src/app/client/src/app/modules/learn/components/course-consumption/assessment-player/assessment-player.component.ts

```diff
@@ -118,6 +118,7 @@ export class AssessmentPlayerComponent implements OnInit, OnDestroy, ComponentCa
   contentRatingModal = false;
   isDesktopApp= false;
   isConnected= true;
+  assessmenType:any;
```

Comment:
@sohailamjad12 rename variable name to isPIAA (boolean variable) as you are saving boolean value to it not assessmentType (PIAA assessment, Self assessment etc);